pkg/kubeapiserver/options/plugins.go

```diff
@@ -143,11 +141,8 @@ func DefaultOffAdmissionPlugins() sets.String {
 		storageobjectinuseprotection.PluginName, //StorageObjectInUseProtection
 		podpriority.PluginName,                  //PodPriority
 		nodetaint.PluginName,                    //TaintNodesByCondition
+		runtimeclass.PluginName,                 //RuntimeClass, gates internally on the feature
 	)
 
-	if utilfeature.DefaultFeatureGate.Enabled(features.RuntimeClass) {
```

Comment:
this wasn't doing what we thought... the `--feature-gates` flag hasn't been parsed at this point :-/